torch/_utils.py

```diff
@@ -138,6 +138,15 @@ def _rebuild_tensor_v2(storage, storage_offset, size, stride, requires_grad, bac
     tensor._backward_hooks = backward_hooks
     return tensor
 
+def _rebuild_qtensor(storage, storage_offset, size, stride, scale, zero_point, requires_grad, backward_hooks):
```

Comment:
how would this need to change with per-channel quantization? because you are hard-coding scale/zero_point here

Reply:
we should save qscheme as well, this will be added after qscheme diff is landed. and we'll call different qtensor creation functions depending on qscheme. we'll probably represent quantization parameters like scale/zp as dictionaries so it is polymorphic to different quantization schemes.